_layouts/post.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+custom_js:
+- prism
+- guides
+---
+
+<div>
+  <div class="container-fluid section-ghostwhite">
+    <div class="col-md-3">
+      <div id="all-guides">
+        <label class="pull-right" >
+          <a href="/guides">&lt; All Guides</a>
+        </label>
+      </div>
+        <div id="toc">
+            {% toc %} {{page.document | tocify_asciidoc}}
+        </div>
+    </div>
+    <div class="col-md-7 post-section-white">
+        <div class="col-md-9 col-md-offset-2 post-title text-center">
+            <img src="{{ page.image }}" alt="post logo">
+            <label>{{ page.title }}</label>
+            <p>
+                {% assign words = content | number_of_words %}
+                {% if words < 360 %}
+                  1 min read.
+                {% else %}
+                  {{ words | divided_by:180 }} mins read.
+                {% endif %} 
+                Updated {{ page.date | date_to_string }}
+            </p>
+            {% if page.tags %}
+              {% for tag in page.tags %}
+                <span class="badge badge-pill badge-default">
+                  {% capture tag-name %}{{tag | upcase }}{% endcapture %}
+                  {% if tag-name=="AWS" or tag-name=="GCP"  or tag-name=="GKE" %} {{ tag-name }} {% else %} {{tag-name | capitalize}}{% endif %}</span>
+              {% endfor %}
+            {% endif %}
+          </div>
+          <div class="content">
+              {{ content }}
+          </div>
+
+    </div>
+    <div class="col-md-2">
+        <div class="card contact-us-card text-center">
+          <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+            <div class="card-body">
+              <p class="card-title">Want to learn more? Feel free to contact us for support</p>
+              <a href="#" class="btn btn-primary">Support</a>
+              <a href="#">Dismiss</a>
```

Comment:
Where does this support upsell show up? Why are the `href` not filled in?

Reply:
It shows up on the side 
<img width="510" alt="Screenshot 2019-08-21 at 12 43 20 PM" src="https://user-images.githubusercontent.com/26963758/63429199-55886100-c411-11e9-9a9a-7ad9cc5bc648.png">